diracx-routers/src/diracx/routers/jobs/submission.py

```diff
@@ -0,0 +1,204 @@
+from __future__ import annotations
+
+import asyncio
+import logging
+from datetime import datetime, timezone
+from http import HTTPStatus
+from typing import Annotated
+
+from fastapi import Body, Depends, HTTPException
+from pydantic import BaseModel
+from typing_extensions import TypedDict
+
+from diracx.core.models import (
+    JobStatus,
+)
+from diracx.core.properties import JOB_ADMINISTRATOR, NORMAL_USER
+
+from ..auth import has_properties
+from ..dependencies import (
+    JobDB,
+    JobLoggingDB,
+)
+from ..fastapi_classes import DiracxRouter
+from ..utils.users import AuthorizedUserInfo, verify_dirac_access_token
+from .access_policies import ActionType, CheckWMSPolicyCallable
+
+logger = logging.getLogger(__name__)
+
+router = DiracxRouter(dependencies=[has_properties(NORMAL_USER | JOB_ADMINISTRATOR)])
```

Comment:
Same here, do we still have to include the `dependencies` parameter here now that we have the access policies?

```suggestion
router = DiracxRouter()
```